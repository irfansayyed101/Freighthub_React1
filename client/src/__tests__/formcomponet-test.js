import React from 'react';
import { shallow } from 'enzyme';
import FormComponent from '../components/FormComponent';

describe('<FormComponent /> with props ' , () =>{
    let shipments =  [{
        "id": "S1000",
        "name": "T2-Shirts(Summer 2018) from Shanghai to Hamburg",
        "cargo": [
          {
            "type": "Fabric",
            "description": "1000 Blue T-shirts",
            "volume": "2"
          },
          {
            "type": "Fabric",
            "description": "2000 Green T-shirts",
            "volume": "3"
          }
        ],
        "mode": "sea",
        "type": "FCL",
        "destination": "Saarbr√ºcker Str. 38, 10405 Berlin",
        "origin": "Shanghai Port",
        "services": [
          {
            "type": "customs"
          }
        ],
        "total": "1000",
        "status": "ACTIVE",
        "userId": "U1000"
      }]
    const container = shallow(<FormComponent shipmentDetail={shipments}/>);
    it('should match the snapshot', () => {
      expect(container.html()).toMatchSnapshot();
    });
});