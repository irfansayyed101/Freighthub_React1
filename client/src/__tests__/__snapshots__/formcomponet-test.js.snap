// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormComponent /> with props  should match the snapshot 1`] = `"<div class=\\"container containerSty\\"><form><div class=\\"form-group row\\"><label for=\\"name\\" class=\\"col-sm-2 col-form-label\\">Name</label><div class=\\"col-sm-4\\"><input type=\\"text\\" class=\\"form-control\\" name=\\"name\\"/></div></div><div class=\\"form-group row\\"><label for=\\"mode\\" class=\\"col-sm-2 col-form-label\\">Mode</label><div class=\\"col-sm-4\\"><label type=\\"text\\" name=\\"mode\\"></label></div></div><div class=\\"form-group row\\"><label for=\\"mode\\" class=\\"col-sm-2 col-form-label\\">Type</label><div class=\\"col-sm-4\\"><label type=\\"text\\" name=\\"type\\"></label></div></div><div class=\\"form-group row\\"><label for=\\"mode\\" class=\\"col-sm-2 col-form-label\\">Destination</label><div class=\\"col-sm-4\\"><label type=\\"text\\" name=\\"destination\\"></label></div></div><div class=\\"form-group row\\"><label for=\\"mode\\" class=\\"col-sm-2 col-form-label\\">Origin</label><div class=\\"col-sm-4\\"><label type=\\"text\\" name=\\"total\\"></label></div></div><div class=\\"form-group row\\"><label for=\\"mode\\" class=\\"col-sm-2 col-form-label\\">Total</label><div class=\\"col-sm-4\\"><label type=\\"text\\" name=\\"total\\"></label></div></div><div class=\\"form-group row\\"><label for=\\"mode\\" class=\\"col-sm-2 col-form-label\\">Status</label><div class=\\"col-sm-4\\"><label type=\\"text\\" name=\\"status\\"></label></div></div><div class=\\"form-group row\\"><div class=\\"col-sm-3 offset-sm-0\\"><input type=\\"submit\\" value=\\"Submit\\"/></div><input type=\\"button\\" class=\\"backButton\\" value=\\"Go Back To Summary Page!\\"/></div></form></div>"`;
